src/layman/rest_publication_test.py

```diff
@@ -58,9 +53,9 @@ class TestSoapClass:
     access_rights_rewo = {'read': f"{username},EVERYONE", 'write': f"{username}"}
     access_rights_rewe = {'read': f"{username},EVERYONE", 'write': f"{username},EVERYONE"}
 
-    @pytest.fixture(scope='class')
+    @pytest.fixture()
```

Comment:
Why did you remove scope="class"?

Reply:
Reverted. Solved by setting scope="class" for ensure_layman fixture.